README.md

```diff
@@ -96,5 +104,5 @@ KEYCLOAK_URL=https://keycloak.dev.callisto-notprod.homeoffice.gov.uk KEYCLOAK_CL
 
 Access key and secret key for the terraform s3 bucket can be found in Kubernetes secrets.
 
-You will also need to get the client secret (found in the credentials tab) from the terraform-client in the Callisto 
-realm in keycloak, and update the `KEYCLOAK_CLIENT_SECRET` in Drone.
\ No newline at end of file
+You will also need to get the client secret (found in the credentials tab) from the terraform-client in the Callisto
+realm in keycloak, and update the `KEYCLOAK_CLIENT_SECRET` in Drone.
```

Comment:
Do we need the all Callisto repos section?

Reply:
added now